src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+
+    private int firstSide;
+    private int secondSide;
+    private int height;
+
+    public IsoscelesTrapezoid(String color, int firstSide, int secondSide, int height) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        this.height = height;
+    }
+
+    @Override
+    public double area() {
+        return (double) (firstSide * secondSide / 2) * height;
```

Comment:
The formula for the area of an isosceles trapezoid should be `((firstSide + secondSide) / 2.0) * height` instead of `(firstSide * secondSide / 2) * height`. The current formula is incorrect and will not yield the correct area.